packages/forma-36-react-components/src/components/Illustration/README.mdx

```diff
@@ -5,10 +5,32 @@ status: 'stable'
 slug: /components/illustration/
 github: 'https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components/src/components/Illustration'
 storybook: 'https://f36-storybook.contentful.com/?path=/story/components-iconbutton--default'
+typescript: ./Illustration.tsx
 ---
 
-Detailed documentation will be provided soon. For now, please have a look into the storybook for implementation details.
+Illustration is a component that helps convey complex ideas in a graphic way.
+
+## Table of contents
+
+- [How to use Illustration](#how-to-use-empty-state)
+- [Code examples](#code-examples)
+- [Props](#props)
+
+## How to use Illustration
+
+Use Illustration when your application needs a graphic enhancement / aid.
+It can be used as a standalone component or alongside text.
```

Comment:
Maybe since we mention the usage of the component alongside text, it's worth adding a code example for it? WDYT?

Reply:
sure, I can do that :)